app/constants/teacherStyle.ts

```
@@ -0,0 +1,19 @@
+
+export const TEACHER_STYLE_TEXT = {
+  kind: "따뜻하고 친절한",
+  passionate: "열정적인",
+  meticulous: "꼼꼼한",
+  funny: "재미있는",
+  selfEsteem: "자존감 지킴이",
+  skillful: "수업 중심을 잡아주는 능숙한",
+};
+
+export const TEACHER_STYLE_ICON = {
+  kind: solar_fire,
```

ESLint

[error] Identifier 'solar_fire' is not in camel case.

(camelcase)